packages/core-sdk/src/types/common.ts

```diff
@@ -1,4 +1,18 @@
+import { Address, WalletClient } from "viem";
+
+import { SetPermissionsRequest } from "./resources/permission";
+
 export type TypedData = {
   interface: string; // i.e. "(address,uint256)"
   data: unknown[];
 };
+
+export type SignatureHelpParameter = {
```

Comment:
How is this parameter used? The naming is a bit weird. Can it just be SignatureParameter?

Reply:
I think it should just be getPermissionSignatureRequest